src/appleseed-max-impl/appleseedrenderer/projectbuilder.cpp

```diff
@@ -573,6 +630,10 @@ namespace
                 for (int i = 0; i < submtlcount; ++i)
                 {
                     Mtl* submtl = mtl->GetSubMtl(i);
+
+                    if (type == RenderType::MaterialPreview)
```

Comment:
Why in other places below are you checking that `type` is **not** equal to `RenderType::MaterialPreview`, and here you're doing the opposite?

Reply:
Great catch @dictoon , it's a bug. Thanks!